plugins/identity/app/javascript/widgets/app_credentials/List.jsx

```diff
@@ -48,7 +48,7 @@ const AppCredentialsList = ({ userId, refreshRequestedAt, projectId, overlay })
         setAppCredentialsFoundForProject(foundProject)
       })
       .catch((error) => {
-        setError(error.message)
+        setError(error.data.error.error.message)
```

Comment:
Actually with this deeply nested object you should do the conditional query, to not run into runtime errors if the object doesn't look like this and for good measure add a fallback error message:
```suggestion
        setError(error?.data?.error?.error?.message || "Something went wrong" )
```